core/internal/settings/settings.go

```diff
@@ -63,6 +63,11 @@ func (s *Settings) IsSync() bool {
 	return s.Proto.XSync.GetValue()
 }
 
+// The path to the sync file.
```

Comment:
This might be a good place to put (a little) more detail about what the sync file is.

Reply:
good point... maybe we should rename it to transaction log?

Reply:
actually yes, that would be good! I didn't even realize this was referring to the transaction log